rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :data, :person, :book
+
```

Comment:
- [ ] 😀 Please modify the spelling of date and n to data. That's a typo but it will cause the actual implementation expected to not work. 👍🏽